lib/hscode/input_parser.rb

```diff
@@ -62,12 +69,15 @@ def list_status_codes(opts)
 
     def display_help_message(opts)
       opts.on_tail('-h', '--help', 'Show this help message') do
-        puts opts, 'Examples:
+        puts opts, "Examples:
           hscode -c 200
           hscode -c 200 -v
           hscode -l
+          hscode -s ok
```

Comment:
I'd recommend that we have search examples come after "list descriptions" (`hscode -l 2xx`) for consistency.